src/style.css

```diff
@@ -0,0 +1,122 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&family=Poppins:wght@300;400;600;800&display=swap');
+
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background-color: #f2f2f2;
+  font-family: 'Poppins', sans-serif;
+  color: #555;
+}
+
+main {
+  width: 500px;
+  max-width: 90%;
+  margin: 150px auto;
+  background-color: white;
+  box-shadow: rgba(0, 0, 0, 0.1) 0 4px 12px;
+}
+
+.main-item {
+  padding: 15px;
+  border-bottom: 1px solid #b6b6b6;
+}
+
+.caption {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.icon {
+  color: #b6b6b6;
+  font-size: 20px;
+  cursor: pointer;
+}
+
+.form-main {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+input {
+  border: none;
+  outline: none;
+  font-size: 15px;
+  width: 100%;
+  font-family: 'Poppins', sans-serif;
+  margin-left: 4px;
+}
+
+.arrow-left {
+  font-size: 12px;
+  color: #b6b6b6;
+}
+
+::placeholder {
+  color: #b6b6b6;
+  font-style: italic;
+}
+
+li {
+  display: flex;
+  align-items: center;
+}
+
+li p {
+  font-size: 14px;
+  margin-left: 5px;
+}
+
+.more {
+  display: block;
+  margin-left: auto;
+  font-size: 25px;
+}
+
+.clear {
+  padding: 20px;
+  text-align: center;
+  background-color: #f2f2f2;
+  cursor: pointer;
+  width: 100%;
+}
+
+button {
+  border: none;
+  color: #9e9e9e;
+  font-size: 16px;
+  cursor: pointer;
+  font-family: 'Poppins', sans-serif;
+}
+
+.hide {
+  display: none;
+}
+
+.edit-background {
+  background-color: rgb(243, 243, 176);
+}
+
+.checkbox {
+  display: inline;
+
+  /* border: 1px solid red; */
+  width: 5%;
+}
+
+.todo-p {
+  /* border: 1px solid red; */
+  width: 100%;
+}
+
+/*
+.li {
+  border: 1px solid red;
+} */
```

Comment:
always remember to check your code for commented code you should not leave commented code.😁😁